### PR TITLE
An option to show the colon at the start of the command line box

### PR DESCRIPTION
**Pull request by gadkadosh**
Just a simple option to show the colon character at the start of the command line box.
Something I've been missing from vim itself, maybe others would find this useful too. If set to false the behavior should be the same like now.
I find it helps distinguish the vim command line with the command palette and the file search boxes.

### CODE REVIEW

**Comment by xconverge** [comment]
I would like to have it set to false by default as to not confuse our current userbase. Also can you add this to the README.md settings area, thanks!

**Comment by gadkadosh** [comment]
Sorry about that, was being stupid. Or just tired.
Fixed also a typo in package.json related to vim.substituteGlobalFlag option. 
I would suggest setting ignoreFocusOut to false for the command line box. The way it is now is often quite a bit in the way, and forces you to use the mouse again to gain focus back, in a very non-vim way.
Should I squash commits together?

**Comment by xconverge** [comment]
Hmm the merge still doesn't look right...can you try again please? Github lets me squash and merge which I can do at the end once it is ready to go

**Comment by gadkadosh** [comment]
What do you mean? It merges fine for me now. Have I messed something up?

File: package.json [hunk 1]
```diff
@@ -550,7 +550,11 @@
         "vim.cmdLineInitialColon": {
           "type": "boolean",
           "description": "When typing a command show the initial colon ':' character",
+<<<<<<< HEAD
```
Comment (jpoon) [inline package.json]: Can you fix these merge conflicts?

File: src/configuration/configuration.ts [hunk 2]
```diff
@@ -328,7 +328,11 @@ class ConfigurationClass {
   /**
    * When typing a command show the initial colon ':' character
    */
+<<<<<<< HEAD
```
Comment (jpoon) [inline src/configuration/configuration.ts]: Unresolved merge conflicts

**Comment by xconverge** [comment]
you didn't modify extension.ts yet it shows up in the diff...

**Comment by gadkadosh** [comment]
I'm sorry, I'm not following. Those snippets seem to me to be from an older commit, not the most recent one. The most recent one really doesn't conflict for me with the master. So at the moment I'm not sure what it is that I should fix. 
Mind you, I'm quite new to this, so maybe I'm just doing something really wrong. I would appreciate if you would point me in the right direction. Thanks!

**Comment by xconverge** [comment]
squashed it and it looks good, thanks for the contribution!

https://github.com/VSCodeVim/Vim/commit/68137e91eb5f8783c08b6f2c7517410e95d7994e

**Comment by gadkadosh** [comment]
Thanks!
I would still like to add ignoreFocusOut false, which seems to be the same like the builtin input boxes of VSCode, and is also more convenient for a vim-like mouse-less usage. What do you think?

**Comment by xconverge** [comment]
sure that sounds good to me!

**Pull request merged**